### DISCUSSION TITLE
Improving your code for style, quality, maintainability, design... with Roslyn Analyzers

### DISCUSSION THREAD

**Discussion by giscus** [piraces/blog#407] [Announcements]
# Improving your code for style, quality, maintainability, design... with Roslyn Analyzers

Learn how Roslyn analyzers analyze your code for style, quality and maintainability, design and other issues.

https://piraces.dev/posts/improve-your-code-using-roslyn-analyzers/

**Comment by udlose** [comment]
might be nice to have an example of adding an operator to the banned list. For example:
`M:System.DateTimeOffset.op_Implicit(System.DateTime)~System.DateTimeOffset;Might use an unexpected UTC offset since it determines timezone offset based on DateTime.Kind value.`
